mscore/timeline.cpp

```diff
@@ -2842,6 +2877,23 @@ QString Timeline::cursorIsOn()
             return "";
       }
 
+//---------------------------------------------------------
+//   updateTimelineTheme
+//---------------------------------------------------------
+
+void  Timeline::updateTimelineTheme()
+      {
+      if(preferences.isThemeDark())
+            _activeTheme = _darkTheme;
+      else
+            _activeTheme = _lightTheme;
```

Comment:
I think it's better if you make a method
```
const TimelineTheme& activeTheme() const { 
      if (preferences.isThemeDark())
            return  _darkTheme;
      else
            return  _lightTheme;
}
```

Reply:
Although it can be so. Only theme getter is still better to change to  
```
const TimelineTheme& activeTheme() const ...
```

Reply:
@igorkorsukov I will change the getter to return a reference, but I don't understand the benefit of changing the theme-accessor. It decreases performance (very slightly, but still) and I believe that it worsens the readability of the code, because as it is it's clear where and when the theme changes. 

If you still believe that this change would be beneficial I will change it, but I would like to learn the reasons (so that I can improve further PRs).

Reply:
The fewer variables that need to be synchronized, the less likely there is error and confusion.

If you have three themes variables, then there may be such problems:   
* `_activeTheme` will not be uninitialized
* `_activeTheme` will not be updated on time
* `_activeTheme` will contain not actual state  
   
These are not problems of this code, these are potential problems of the future when the code changes.
   
You can leave it as it is, but I would recommend doing it everywhere through the getter. Then in the future, it will only be possible to change it without changing another code.

Reply:
Implemented your suggestion.